azure_monitor/src/azure_monitor/export/metrics/__init__.py

```diff
@@ -26,14 +26,20 @@
 
 
 class AzureMonitorMetricsExporter(BaseExporter, MetricsExporter):
+    """Azure Monitor metrics exporter for OpenTelemetry.
+
+    Args:
+        options: Configuration options for the exporter
```

Comment:
Generated docs should have a link to options. Surround "options" with backticks. -> "`options`"